native/src/modules/layout/components/TransparentHeader.js

```diff
@@ -60,13 +59,13 @@ const MaterialHeaderButtons = props => {
 }
 
 class TransparentHeader extends React.PureComponent<PropsType> {
-  goBack = () => {
-    this.props.navigation.goBack(null)
+  goBackInStack = () => {
+    this.props.navigation.goBack()
   }
 
   onShare = async () => {
-    const { navigation, t } = this.props
-    const shareUrl: string = navigation.getParam('shareUrl')
+    const { scene, t } = this.props
+    const shareUrl = scene.route.params?.shareUrl || ''
```

Comment:
I think it makes more sense to pass null if the share path is undefined instead of an empty string.

Reply:
Will fix in #105